"""
Levantando os próprios erros com raise

raise -> Lança exceções

OBS: O raise não é uma função. E uma palavra reservada, assim como def ou qualquer outra em Python.

Para simplificar, pense no raise como sendo útil para que possamos criar nossas próprias exceções e mensagens
de erro.

A forma geral de utilização é:

raise TipoDoError('Mensagem de erro')

OBS: O raise, assim como return, finaliza a função. Ou seja, nada após o raise é executado.
"""